client/state/plans/test/selectors.js

```diff
@@ -51,94 +51,21 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '#getPlanPriceObject()', () => {
-		it( 'should return price plan object', () => {
+	describe( '#getPlanRawPrice()', () => {
+		it( 'should return annual raw price', () => {
 			const state = getStateInstance();
-			const priceObject = getPlanPriceObject( state, 1003 );
-			expect( priceObject ).to.eql( {
-				currencySymbol: '$',
-				decimalMark: '.',
-				dollars: 99,
-				cents: 0
-			} );
+			const price = getPlanRawPrice( state, 1003 );
+			expect( price ).to.eql( 99 );
```

Comment:
I'm not sure I follow. This is a selector test to verify we're pulling the price data from our redux tree. Prices will always come from the api endpoint.